### PR TITLE
add stacktrace to application error log

### PR DESCRIPTION
**Pull request by pivnicek**
fixes: #28

### CODE REVIEW

File: src/run.php [hunk 1]
```diff
@@ -54,7 +54,14 @@
 
     exit(1);
 } catch (ApplicationException $e) {
-    $logger->log('error', $e->getMessage(), (array) $e->getData());
+    $logger->log('error', $e->getMessage(), array_merge(
```
Comment (Halama) [inline src/run.php]: I think that the context won't be appended to `stderr`

Comment (Halama) [reply src/run.php]: but it is working on redshift ex, i am debugging it

Comment (pivnicek) [reply src/run.php]: I'll check out the redshift ex one, dik!

Comment (Halama) [reply src/run.php]: i am not able to simulate this kind of error `"errorOutput": "Undefined index: Mpsexp.Parcel"`

Comment (pivnicek) [reply src/run.php]: ah, I don't think it is caught at all.  Undefined index is probable ErrorException

Comment (pivnicek) [reply src/run.php]: And this extractor just catches Exception

Comment (Halama) [reply src/run.php]: it is correctly converted to exception and catched

Comment (Halama) [reply src/run.php]: image

Comment (Halama) [reply src/run.php]: And this is output executed in production image of extractor:
```
[root@f62f49b96b8a src]# php run.php --data /data
file_get_contents(/config.yml): failed to open stream: No such file or directory
```

Comment (Halama) [reply src/run.php]: And there is wrong formatter for errors:
image

Comment (Halama) [reply src/run.php]: omg, i had wrong composer packages installed locally

Comment (Halama) [reply src/run.php]: this is wrong https://github.com/keboola/db-extractor-common/blob/7ea835114452f7d94fa799bf3551fa2c6b92fc7c/src/Keboola/DbExtractor/Logger.php#L28

Comment (pivnicek) [reply src/run.php]: omg :-(

Comment (pivnicek) [reply src/run.php]: jo, I saw that, 👍

**Pull request merged**